nomad/nextcloud.nomad

```diff
@@ -0,0 +1,261 @@
+job "nextcloud" {
+  datacenters = ["dc1"]
+  type = "service"
+  
+  group "nc" {
+    task "nextcloud" {
+      driver = "docker"
+      config {
+        image = "nextcloud:15"
+        volumes = [
+          "/opt/hoover/volumes/nextcloud:/var/www/html",
+          "/opt/hoover/collections/ncsync:/var/www/html/data/ncsync/files",
+        ]
+        port_map {
+          http = 80
+        }
+        labels {
+          liquid_task = "nextcloud"
+        }
+      }
+      env {
+        POSTGRES_DB: nextcloud
+        POSTGRES_USER: postgres
+        POSTGRES_PASSWORD: secret
+        POSTGRES_HOST: nextcloud-pg
+        OC_PASS: secret
+      }
+      resources {
+        network {
+          port "http" {}
+        }
+      }
+      service {
+        name = "nextcloud"
+        port = "http"
+      }
+      }
+
+    task "pg" {
+      driver = "docker"
+      config {
+        image = "postgres:9.6"
+        volumes = [
+          "/opt/hoover/volumes/nextcloud-pg/data:/var/lib/postgresql/data",
+        ]
+        labels {
+          liquid_task = "nextcloud-pg"
+        }
+        port_map {
+          pg = 5432
+        }
+      }
+      env {
+        POSTGRES_USER = "postgres"
+        POSTGRES_PASSWORD = "secret"
```

Comment:
Let's add `TODO`s so we remember to change the secrets. (we can store them in [vault](https://www.vaultproject.io/))